source/styleEditor.js

```diff
@@ -1,3 +1,20 @@
-$(document).ready(function(){
-  // your code goes here.
-});
+$(document).ready(function () {
+    let selector = $('input[name="selector"]');
```

Comment:
Another thing you could have done is `.val()` for the variables up here to improve readability below.